Tensile/KernelWriterAssembly.py

```diff
@@ -20,10 +20,9 @@
 ################################################################################
 
 from SolutionStructs import DataType
-from Common import globalParameters, printExit, printWarning
+from Common import globalParameters, printExit, printWarning, print1, print2
```

Comment:
Linter gives the following error:

M7IZMN2KPFQYNMDFED4FE2ZJHDBR3F7AMRA4TCOLVKDIIAHSVVMA/src/tensile/Tensile/KernelWriterAssembly.py:23: 'Common.print1' imported but unused

My understanding is that the print1 needs to be removed.